sources/install/package_web.sh

```diff
@@ -922,7 +922,22 @@ function package_web() {
     install_soapui                  # SoapUI is an open-source web service testing application for SOAP and REST
     install_sqlmap                  # SQL injection scanner
     install_sslscan                 # SSL/TLS scanner
+    install_Postman                 # Postman - API platform for testing APIs
```

Comment:
Can you replace `Postman` with `postman` (lowercase) ?

Reply:
Yes, done